client/app/index.js

```diff
@@ -37,3 +37,6 @@ if (!global.Intl) {
 } else {
   loadCurrentModule();
 }
+
+// Initializers
+require('./lib/helpers/confirm_dialog');
```

Comment:
actually same as `loadCurrentModule`, both places need to have. 

Otherwise it will break for browsers which do not support intl.